build.gradle

```diff
@@ -16,6 +16,10 @@ plugins {
 
 repositories {
     mavenCentral()
+
+    maven {
+        url "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/" // for intel deflater snapshots
```

Comment:
Looking at the POM for this dependency (https://artifactory.broadinstitute.org/artifactory/libs-snapshot/com/intel/intel-gkl/0.0.1-SNAPSHOT/intel-gkl-0.0.1-20160524.180210-1.pom), I see it uses the `gatk-native-bindings` dependency.  Is  that something we want included in htsjdk?  It looks fairly light-weight, but looks like it is meant for native calls for the HMM in GATK's haplotype caller.